app/controllers/export_controller.rb

```diff
@@ -1,11 +1,23 @@
 class ExportController < AuthenticatedController
   include ProjectScoped
 
+  NON_DRAFT_STATES = ['published', 'ready_for_review'].freeze
+
   def index
+    @default_button_state = non_draft_records? ? 'published' : 'all'
   end
 
   private
 
+  def non_draft_records?
+    issue_states = current_project.issues.pluck(:state).uniq
+    (issue_states & NON_DRAFT_STATES).any? || non_draft_records_pro
```

Comment:
Can we just check with:
```
Issue.published.or(Issue.ready_for_review).any?
```

this way, we don't have to fetch all issue states and iterate through each